src/main/frontend/components/header.cljs

```diff
@@ -189,12 +189,12 @@
                               (state/pub-event! [:go/search]))}
               (ui/icon "search" {:size ui/icon-size})]))])
       (when (mobile-util/native-platform?)
-        (if (or (state/home?) custom-home-page?)
-          left-menu
-          (ui/with-shortcut :go/backward "bottom"
-            [:button.it.navigation.nav-left.button.icon.opacity-70
-             {:title "Go back" :on-click #(js/window.history.back)}
-             (ui/icon "chevron-left" {:size 26})])))]
+        [left-menu
+         (when-not (or (state/home?) custom-home-page? (state/whiteboard-dashboard?))
+           (ui/with-shortcut :go/backward "bottom"
+             [:button.it.navigation.nav-left.button.icon.opacity-70
+              {:title "Go back" :on-click #(js/window.history.back)}
+              (ui/icon "chevron-left" {:size 26})]))])]
 
```

Comment:
I think we can now merge this with the block above
``` 
[left-menu
  (if (mobile-util/native-platform?)
    (when-not (or (state/home?)  ...)
    (when current-repo ...)]
```

Reply:
Done